src/components/EmojiPicker/CategoryShortcutButton.tsx

```diff
@@ -11,19 +9,21 @@ import useThemeStyles from '@hooks/useThemeStyles';
 import getButtonState from '@libs/getButtonState';
 import variables from '@styles/variables';
 import CONST from '@src/CONST';
+import type {TranslationPaths} from '@src/languages/types';
+import type IconAsset from '@src/types/utils/IconAsset';
 
-const propTypes = {
+type CategoryShortcutButtonProps = {
     /** The emoji code of the category header */
-    code: PropTypes.string.isRequired,
+    code: string;
```

Comment:
Let's use a literal string here instead of `string`, values should be based on emojiPicker translations:
```ts
headers: {
            frequentlyUsed: 'Frequently Used',
            smileysAndEmotion: 'Smileys & Emotion',
            peopleAndBody: 'People & Body',
            animalsAndNature: 'Animals & Nature',
            foodAndDrink: 'Food & Drinks',
            travelAndPlaces: 'Travel & Places',
            activities: 'Activities',
            objects: 'Objects',
            symbols: 'Symbols',
            flags: 'Flags',
        },
```

Reply:
After looking into it I'm fine with code being a string (as it was before my suggestion). Sorry!